stories/20-Collapse.stories.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react'
```

Comment:
I added the missing comma above, but I get the following error in storybook 

<img width="1330" alt="Screen Shot 2020-03-16 at 3 21 32 PM" src="https://user-images.githubusercontent.com/32427979/76804643-08e3e900-679a-11ea-94e0-71be1416c8a2.png">